apps/nextjs-example/components/AlertProvider.tsx

```diff
@@ -4,16 +4,16 @@ import {
   FC,
   ReactNode,
   SetStateAction,
+  useCallback,
   useContext,
   useState,
 } from "react";
 import { ErrorAlert, SuccessAlert } from "./Alert";
 
 interface AlertContextState {
-  successAlertMessage: string | null;
-  setSuccessAlertMessage: Dispatch<SetStateAction<string | null>>;
-  errorAlertMessage: string | null;
-  setErrorAlertMessage: Dispatch<SetStateAction<string | null>>;
+  setSuccessAlertHash: (hash: string, networkName?: string) => void;
+  setSuccessAlertMessage: Dispatch<SetStateAction<ReactNode | null>>;
```

Comment:
the only difference in `setSuccessAlertMessage` and `setSuccessAlertHash` is the message content? can't we have one setSuccess function and that would be responsible on the content we want to display?

Reply:
we can probably develop it better but this is just a demo app so should be good for it